doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/user-guides/autoscaling.md

```diff
@@ -0,0 +1,91 @@
+10 Ray pods will be created to satisfy the 100-CPU resource demand.
+In this respect, the Ray autoscaler is similar to the
+[Kuberentes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler),
+which makes scaling decisions based on the logical resources expressed in container
+resource requests.
+## Fine-grained control of scale-down
+To accommodate the statefulness of Ray applications, the Ray autoscaler has more
+fine-grained control over scale-down than the Horizontal Pod Autoscaler. In addition to
+determining desired scale, the Ray Autoscaler is able to select precisely which pods
+to scale down. The KubeRay operator then deletes that pod.
+By contrast, the Horizontal Pod Autoscaler can only decrease a replica count, without much
+control over which pods are deleted. For a Ray application, downscaling a random
+pod could be dangerous.
+## Architecture: One Ray Autoscaler per Ray Cluster.
+Horizontal Pod Autoscaling is centrally controlled by a manager in the Kubernetes control plane;
+the manager controls the scale of many Kubernetes objects.
+By contrast, each Ray cluster is managed by its own Ray autoscaler process,
+running as a sidecar container in the Ray head pod. This design choice is motivated
+by considerations of scalability.
+
+### Ray Autoscaler with Kubernetes Cluster Autoscaler
+The Ray Autoscaler and the
+[Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) complement each other.
+After the Ray autoscaler decides to create a Ray pod, the Kubernetes Cluster Autoscaler
+can scale up a Kubernetes Node so that the pod can be placed.
+Similarly, after the Ray autoscaler decides to delete an idle pod, the Kubernetes
+Cluster Autoscaler can clean up the idle Kubernetes Node that remains.
+It is recommended to configure your RayCluster so that only one Ray pod fits per Kubernetes Node.
+If you follow this pattern, Ray autoscaler Pod scaling events will correspond roughly one-to-one with cluster autoscaler
+Node scaling events. (We say "roughly" because it is possible for a Ray pod be deleted and replaced
+with a new Ray pod before the underlying Kubernetes Node is scaled down.)
+
+
+### Vertical Pod Autoscaler
+There is no relationship between the Ray Autoscaler and the Kubernetes
+[Vertical Pod Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) (VPA),
+which is meant to size individual pods to the appropriate size based on current and past usage.
+If you find that the load on your individual Ray pods is too high, there are a number
+of manual techniques to decrease the load.
+One method is to schedule fewer tasks/actors per node by increasing the resource
+requirements specified in the `ray.remote` annotation.
+For example, changing `@ray.remote(num_cpus=2)` to `@ray.remote(num_cpus=4)`
+will halve the quantity of that task or actor that can fit in a given Ray pod.
```

Comment:
in the latest rendering this still looks like
<img width="785" alt="Screen Shot 2022-08-05 at 2 59 32 AM" src="https://user-images.githubusercontent.com/113316/183053857-ef4ac59f-a360-4a4b-8759-d8655f94c4dd.png">
with double "will", not sure what i going on here

Reply:
hmm, it might not have re-rendered

Reply:
let's see if it re-renders

Reply:
It has rendered correctly now.